### PR TITLE
ingress removed

### PR DESCRIPTION
**Pull request by manojn97**
# Pull Request Review

## Checklist

- [x] Issue: #86 
- [x] List of related PRs. : None
- [x] Project builds without any errors.
- [x] Unit Tests updated (or created where needed) and all pass.
- [x] You have ran manual functional “smoke” test (does product as a whole still work?).
- [x] User Story meets the acceptance criteria - and passes. acceptance criteria should be understood at outset.
- [x] There is a PR for update of internal documentation for the affected Project(s) - README etc.
- [x]  **Description of the changes**:  #86

### CODE REVIEW

**Review by hhellyer — APPROVED** [review]

File: tests/unit/kubeobjects/test_config_doc.py [hunk 1]
```diff
@@ -17,46 +17,6 @@ def test_read_single_doc(self):
         self.assertEqual(len(objects), 1)
         self.assertEqual(objects[0].kind, 'ConfigMap')
 
-    def test_read_multi_doc(self):
```
[the remark this answers is not in the document]
Comment (manojn97) [reply tests/unit/kubeobjects/test_config_doc.py]: Hi @dvaccarosenna  I have updated the PR to not remove any tests cases as they are still functional

File: tests/unit/kubeobjects/example_object_config_files/multidoc_separator_at_end.yaml [hunk 2]
```diff
@@ -1,23 +0,0 @@
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
```
[the remark this answers is not in the document]
Comment (manojn97) [reply tests/unit/kubeobjects/example_object_config_files/multidoc_separator_at_end.yaml]: I have updated the PR to not remove any tests cases as they are still functional

File: tests/unit/kubeobjects/example_object_config_files/multidoc_emptydoc_in_middle.yaml [hunk 3]
```diff
@@ -1,23 +0,0 @@
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: testing
```
[the remark this answers is not in the document]
Comment (manojn97) [reply tests/unit/kubeobjects/example_object_config_files/multidoc_emptydoc_in_middle.yaml]: I have updated the PR to not remove any tests cases as they are still functional

File: tests/unit/kubeobjects/example_object_config_files/multidoc_dots_at_end.yaml [hunk 4]
```diff
@@ -1,23 +0,0 @@
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: testing
```
[the remark this answers is not in the document]
Comment (manojn97) [reply tests/unit/kubeobjects/example_object_config_files/multidoc_dots_at_end.yaml]: I have updated the PR to not remove any tests cases as they are still functional

File: docs/user-guide/ready-checks.md [hunk 5]
```diff
@@ -196,28 +184,8 @@ When `found` is `True`, the value of `helm_release` will be a special Helm objec
...
-  ...
-}
-```
-
 The `objects` attribute has the same functions as `keg.objects`, so you can lookup objects in the Helm release by their apiVersion, kind, name and namespace.
 
-```python
-def checkReady(keg, props, resultBuilder, log, *args, **kwargs):
-    found, helm_release = keg.helm_releases.get('MyReleaseName', 'MyNamespace')
-    if found:
```
[the remark this answers is not in the document]
Comment (manojn97) [reply docs/user-guide/ready-checks.md]: I have made sure to undo this change now

File: docs/user-guide/ready-checks.md [hunk 6]
```diff
@@ -196,28 +184,8 @@ When `found` is `True`, the value of `helm_release` will be a special Helm objec
 `info` - dictionary of details related to the Helm release
 `objects` - a collection of the objects managed by this Helm release
 
-Example of `info`:
-
-```python
```
[the remark this answers is not in the document]
Comment (manojn97) [reply docs/user-guide/ready-checks.md]: I have reverted this change

File: docs/user-guide/building-a-resource.md [hunk 7]
```diff
@@ -345,7 +345,7 @@ compose:
           name: my-release-name
 ```
 
-The value of `chart` must be either a file in the `helm` directory of your Resource’s Kubernetes directory or the name of a chart from a repository (that has been configured on the Helm server of your deployment location) e.g. `stable/nginx-ingress`. 
+The value of `chart` must be either a file in the `helm` directory of your Resource’s Kubernetes directory or the name of a chart from a repository (that has been configured on the Helm server of your deployment location)
```
[the remark this answers is not in the document]
Comment (manojn97) [reply docs/user-guide/building-a-resource.md]: I have reverted this change

File: docs/reference/associated-topology.md [hunk 8]
```diff
@@ -133,58 +133,4 @@ Example with removed object named `example`:
 ```
 associatedTopology:
   v1:ConfigMap:doc:example: null
-```
-
-Example with deployed Helm release named `example`:
```
[the remark this answers is not in the document]
Comment (manojn97) [reply docs/reference/associated-topology.md]: I have reverted this change

**Review by hhellyer — APPROVED** [review]
Approving as these changes look smaller and reasonable but @dvaccarosenna is better placed to say whether they now meet the requirements.

**Review by dvaccarosenna — APPROVED** [review]

**Pull request merged**